eng/CredScanSuppression.json

```diff
@@ -2,19 +2,12 @@
-      ],
-      "_justification": "Test Playback certificates."
+      "file": [ "sdk/attestation/azure-security-attestation/test/ut/crypto_test.cpp" ],
```

Comment:
We typically avoid entire file exclusions unless the entire file a fake like the cert file. If you want to have these be in a file please move the fake secrets to a file and then name the file like fake_test_secrets.cpp/h or something like that so it is clear that all the things in there are fake.

Reply:
Fixed. 

FWIW, these are *NOT* fake secrets. 

They are very much real secrets (otherwise the test wouldn't work).

This is a 100% allowable exception to the credscan requirements because these secrets are not actually used to encrypt any data - their purpose is to test the logic to read various types of secrets.

Reply:
By real do you mean, they were functional at some point in the past? Do we have to occasionally update these real secrets?